client/app/scripts/components/node-details/node-details-table.js

```diff
@@ -169,10 +169,12 @@ export default class NodeDetailsTable extends React.Component {
           <tbody>
           {nodes && nodes.map(node => {
             const values = this.renderValues(node);
+            const nodeId = node.node_id || node.id;
```

Comment:
There's probably a `table` that both node details table and connections table "inherit" from.